src/pages/home/report/ReportActionsList.tsx

```diff
@@ -466,7 +466,7 @@ function ReportActionsList({
         (isFromCurrentUser: boolean) => {
             // If a new comment is added and it's from the current user scroll to the bottom otherwise leave the user positioned where
             // they are now in the list.
-            if (!isFromCurrentUser || scrollingVerticalOffset.current === 0 || !isReportScreenTopmostCentralPane()) {
+            if (!isFromCurrentUser || !isReportScreenTopmostCentralPane()) {
```

Comment:
Removed the condition. It's just for early returns based on this [comment](https://github.com/Expensify/App/pull/54071#discussion_r1900885689).